server/aws-lsp-codewhisperer/src/language-server/netTransform/utils.ts

```diff
@@ -0,0 +1,158 @@
+import * as archiver from 'archiver'
+import * as crypto from 'crypto'
+import * as fs from 'fs'
+import * as path from 'path'
```

Comment:
I don't think any of these are cross platform (node, browser, etc). We should not import dependencies that will limit the whether this server or components of it can be bundled to target different runtimes.

Reply:
We have lot of usage of `fs` I dont see everything available in runtime. For example for `fs.writeFileSync` , this is something needed for our functionality